easybuild/easyconfigs/n/ncurses/ncurses-6.3-GCCcore-12.2.0.eb

```
@@ -0,0 +1,47 @@
+easyblock = 'ConfigureMake'
+
+name = 'ncurses'
+version = '6.3'
+
+homepage = 'https://www.gnu.org/software/ncurses/'
+description = """
+ The Ncurses (new curses) library is a free software emulation of curses in
+ System V Release 4.0, and more. It uses Terminfo format, supports pads and
+ color and multiple highlights and forms characters and function-key mapping,
+ and has all the other SYSV-curses enhancements over BSD Curses.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '12.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = [GNU_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['97fc51ac2b085d4cde31ef4d2c3122c21abc217e9090a43a30fc5ec21684e059']
+
+builddependencies = [('binutils', '2.39')]
+
+local_common_configopts = "--with-shared --enable-overwrite --without-ada --enable-symlinks "
```

I am seeing the issue https://github.com/easybuilders/easybuild-easyconfigs/issues/16014 here as well. As in https://github.com/easybuilders/easybuild-easyconfigs/pull/16270 we need to add `--with-versioned-syms`
```suggestion
local_common_configopts = "--with-shared --enable-overwrite --without-ada --enable-symlinks --with-versioned-syms "
```